src/components/Features/Faq/Faq.module.scss

```diff
@@ -0,0 +1,87 @@
+@import '../../../index';
+
+$color: $--color-white;
+$link-color: $--color-gamma;
+$link-color-hover: $--color-gamma-1;
+$visited-link-color: $--color-alpha-6;
+$visited-link-color-hover: $--color-alpha-6-hover;
+$border-color: $--color-alpha-3;
+$background-answer: $--color-alpha-3;
+
+.faq {
+  color: $color;
+  max-height: 500px;
+  overflow-y: auto;
+  text-align: left;
+
+  a {
+    color: $link-color;
+    &:hover {
```

Comment:
add space

Reply:
it's not visible here- but I did add a space, if looking in the src or checking out to this branch it is visible